mediapipe/tasks/ios/MediaPipeTaskCommon.podspec.template

```diff
@@ -0,0 +1,23 @@
+Pod::Spec.new do |s|
+  s.name             = 'MediaPipeTaskCommon'
```

Comment:
Can you change the names to Tasks (with an s)? We use "tasks" for Maven and NPM.

Reply:
Updated names to MediaPipeTasksCommon, MediaPipeTasksVision, MediaPipeTasksText